CONTRIBUTING_TUTORIAL.md

```diff
@@ -0,0 +1,56 @@
+
+## Step 3: Clone your branch.
+For this step, you need the URL of your fork and the name of your branch. Find the directory where you want to store the source code of the game. Then, open a terminal to that directory. Run `git clone <your fork url> -b <your branch name>`. For example, I might run `git clone https://github.com/TestAccount/secret-hitler -b TestAccount-patch1`. 
+
+## Step 4: Install dependencies and run code.
+Change directory into your clone (`cd secret-hitler` on all operating systems). Run `yarn` to install the dependencies. Then, run `yarn dev`. A bunch of lines of debug should output. Wait for it to tell you `Webpack compiled in <some number> seconds` (may take more than 30 seconds on some computers), then open `localhost:8080` in your browser. The site should load! When entering the game lobby, you'll see a bunch of buttons to log in to testing accounts. The accounts don't exist yet. Open a new terminal (don't close the one you already have or the site will close) and run `yarn create-accounts` in the same directory you ran `yarn dev`. If that works, run `yarn assign-local-mod`. Now, Uther is an admin and everything is working!
+
+### Issues
+If you get an issue like `yarn: command not found` when running `yarn dev`, make sure `yarn` is on your PATH. This is a pretty common issue with some installations. If you are on Windows and can't get `yarn` to run, try following [their installation instructions](https://yarnpkg.com/getting-started/install).
+
+## Step 5: Make your changes!
+I can't really help with this step, but now, you have all the code. 
+## Step 6: Push to your branch.
+Make sure you know what files you changed. For each file you changed (or the directory with all the files), run `git add <file>` (from any folder in the main folder of the code). Next, you want to explain what you did. Come up with a message. Usually a sentence or two is enough. **Important**: if you are fixing an issue, make sure to write "fixes #\<issue number>". Then run `git commit -m "<your message>"`. Finally, run `git push`. 
+## Step 7: Make a pull request!
+This is the last step! Go to your branch on GitHub (if you never closed your fork on GitHub, you should be still on it). Go to https://github.com/cozuya/secret-hitler/compare/master...YOUR_GITHUB_NAME:master (replace "YOUR_GITHUB_NAME" with your actual GitHub username). If you did everything right, you can click ![enter image description here](https://i.ibb.co/LNw5pRm/create-pr.png). Fill out a name and description (remember to include "fixes #\<issue number>" in the description if you fixed an issue). Make sure to check "Allow edits by maintainers". Click "Create Pull Request" and you're done!
```

Comment:
You mention cloning a specific branch on 41, but here you say to compare with forked master. I'd say better bet is to keep the cloning and branching instructions on 41, but here instead of giving a link, show the yellow pop-up on any page of the forked repo when there's a branch with recent changes. 

See ![image](https://www.asmeurer.com/git-workflow/pull.png)